sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessMessageEventArgs.cs

```diff
@@ -214,6 +220,10 @@ await _receiver.AbandonMessageAsync(message, propertiesToModify, cancellationTok
             CancellationToken cancellationToken = default)
         {
             await _receiver.RenewMessageLockAsync(message, cancellationToken).ConfigureAwait(false);
+            if (_receiveActions.Messages.TryGetValue(message, out var cancellationTokenSource))
```

Comment:
This could also be encapsulated into a method on the receive actions similar to the `GetLockLostToken` but an internal one